CMakeLists.txt

```diff
@@ -440,8 +440,8 @@ if (CMAKE_SYSTEM_NAME MATCHES "(SunOS|Solaris)")
 endif ()
 
 if (APPLE AND NOT IOS)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=x86-64 -fvisibility=default")
-  if (NOT OpenSSL_DIR)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=x86-64 -fvisibility=default -std=c++11")
```

Comment:
Hmm, this change is not included in Monero PR 6691 but in 5930. I feel a bit uneasy when someone sneaks in some unrelated change in a cherry-pick, and I like it better if 5930 is cherry-picked beforehand (if it's easy). But okay if you insist.

Reply:
Sorry not trying to be sneaky but I understand why the uneasiness. I will break that up to be more transparent.